apps/costcontrol/locales/costcontrol.en-US.properties

```diff
@@ -102,6 +102,9 @@ long-date-format = %B %d, %Y
 sms-sent = SMS sent
 call-made = Calls made
 settings = Settings
+# LOCALIZATION NOTE (settingsButton): label for settings button. The following
+# string is spoken by screen readers and not shown on the screen.
+settingsButton.ariaLabel = Settings
```

Comment:
Please, to maintain the same coding style,  don't camelize the tag settingsButton (use settings-button.ariaLabel)